src/main/java/com/gradle/Utils.java

```diff
@@ -82,29 +82,12 @@ static String stripPrefix(String prefix, String string) {
         return string.startsWith(prefix) ? string.substring(prefix.length()) : string;
     }
 
-    static String appendIfMissing(String str, String suffix) {
-        return str.endsWith(suffix) ? str : str + suffix;
-    }
-
-    static URI appendPathAndTrailingSlash(URI baseUri, String path) {
-        if (isNotEmpty(path)) {
-            String normalizedBasePath = appendIfMissing(baseUri.getPath(), "/");
-            String normalizedPath = appendIfMissing(stripPrefix("/", path), "/");
-            return baseUri.resolve(normalizedBasePath).resolve(normalizedPath);
-        }
-        return baseUri;
+    static String prependIfMissing(String suffix, String str) {
```

Comment:
The `suffix` parameter should be renamed to `prefix`.

Reply:
+1

Reply:
Done.